src/states/index.js

```diff
@@ -0,0 +1,19 @@
+const BOOKS = [
+  {
+    id: Math.floor(Math.random() * 10000),
+    title: 'Book Test 1',
+    category: 'Action',
+  },
+  {
+    id: Math.floor(Math.random() * 10000),
+    title: 'Book Test 2',
+    category: 'Horror',
+  },
+  {
+    id: Math.floor(Math.random() * 10000),
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Book Test 1', category: 'Action'}
```